### PR TITLE
nomis-api-access-staging: service account for tiller, rename rds

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/01-rbac.yaml [hunk 1]
```diff
@@ -11,3 +11,23 @@ roleRef:
...
+metadata:
+  name: tiller
+  namespace: nomis-api-access-staging
+subjects:
+- kind: ServiceAccount
+  name: tiller
+  namespace: nomis-api-access-staging 
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```
Comment (alkar) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/01-rbac.yaml]: Can you please change this role to simply `admin`?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/resources/rds.tf [hunk 2]
```diff
@@ -2,7 +2,7 @@ variable "cluster_name" {}
 
 variable "cluster_state_bucket" {}
 
-module "nomis-api-access-staging_rds" {
+module "nomis-api-access_rds" {
```
Comment (alkar) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/resources/rds.tf]: Changing the name of the module will re-create the RDS instance. You can check the [`plan`](https://concourse.cloud-platform.service.justice.gov.uk/teams/main/pipelines/plan-environments/jobs/plan-live-1/builds/259) for more detail. Are you happy to proceed with this?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/01-rbac.yaml (hunk 1 above)
Comment (mattops) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/01-rbac.yaml]: Yes no prob.   FYO I took that from the guide here https://user-guide.cloud-platform.service.justice.gov.uk/tasks.html#tiller-rbac-configuration

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/resources/rds.tf (hunk 2 above)
Comment (mattops) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/resources/rds.tf]: Yes that's fine.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/01-rbac.yaml (hunk 1 above)
Comment (alkar) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/01-rbac.yaml]: Thanks for the reference, I'll update.

**Pull request merged**